packages/experimental/Switch/src/Switch.tsx

```diff
@@ -14,9 +40,30 @@ export const Switch = compose<SwitchType>({
     toggleContainer: View,
     onOffText: Text,
   },
-  useRender: (_userProps: SwitchProps) => {
-    return (_final: SwitchProps) => {
-      return null; // Only implemented for Win32
+  useRender: (userProps: SwitchProps, useSlots: UseSlots<SwitchType>) => {
+    const switchInfo = useSwitch(userProps);
+    // grab the styled slots
+    const Slots = useSlots(userProps, (layer) => switchLookup(layer, switchInfo.state, switchInfo.props));
+
+    // now return the handler for finishing render
+    return (final: SwitchProps) => {
+      const { label, offText, onText, labelPosition, ...mergedProps } = mergeProps(switchInfo.props, final);
+      const onOffText = switchInfo.state.toggled ? onText : offText;
+      const displayOnOffText = !!offText || !!onText;
+      const isReduceMotionEnabled = AccessibilityInfo.isReduceMotionEnabled;
+      const thumbAnimation = isReduceMotionEnabled ? { animationClass: 'Ribbon_SwitchThumb' } : null;
```

Comment:
This will only work for win32 - it must be replaced on other endpoints.

Reply:
@rurikoaraki , Correct animation doesn't work for now , finding the best way to handle it.

Reply:
We're using LayoutAnimation for Switch on Android.